cernopendata/modules/fixtures/data/docs/delphi-getting-started/delphi-getting-started.md

```diff
@@ -0,0 +1,229 @@
+
+Please read these before accessing the DELPHI software and data.
+
+## Accessing the software stack
+There are two possible ways to access the software stack.
+
+### Docker
+A docker container is available which ships with all the modules installed. Please take a look [here](docs/delpi-guide-docker)
+
+### CVMFS
+Binaries are also available from CVMFS, for a variety of different Linux flavors. There are no native Windows or Apple ports available for the time being.
+
+This method is convenient if you are running a desktop or a virtual machine with one of the supported Linux flavors on it.
+
+#### Requirements
+You will need to have:
+
+* /cvmfs/delphi.cern.ch mounted
+* /eos/opendata/delphi (or /eos/experiments/delphi) mounted
+* A list of additional packages installed, see https://gitlab.cern.ch/delphi/deployment for more information:
+    * general: tcsh xfonts-100dpi xfonts-75dpi libxfont2
+    * compilers: cmake gcc g++ gfortran
+    * library packages: libx11 libglu1-mesa xutils libmotif r-base xutils libxbae libxaw7 libssl libglew libdlm
+
+Please check the instructions at https://cvmfs.readthedocs.io/en/stable/cpt-quickstart.html and https://eoscta.docs.cern.ch/install/eos/ and https://eos-web.web.cern.ch/eos-web.
+
+#### Initialising the environment:
+For C-Shell (csh, tcsh ), do
+```
+ source /cvmfs/delphi.cern.ch/setup.csh
+```
+
+For Bourne shell (sh, bash, zsh, ...), type
+```
+. /cvmfs/delphi.cern.ch/setup.sh
+```
+If using the container, please use ```/etc/profile.d/delphi.sh``` or ```/etc/profile.d/delphi.csh``` instead.
+
+After sourcing the environment, make sure that the environment variable **DELPHI_DATA_ROOT** points to the tip of the data area. This is specifically important if you copied the data sets you want to use locally. The default is ```/eos/experiment/delphi/castor2015```.
+
+## Documentation
+DELPHI manuals and notes are available from [https://cds.cern.ch/](http://cds.cern.ch/search?ln=en&p=&action_search=Search&op1=a&m1=a&p1=&f1=&c=DELPHI&sf=&so=d&rm=&rg=10&sc=1&of=hb)
+
+Here is a selection for getting started:
+
+* [Analysis framework](/docs/27002)
+* [Event display](/docs/27003)
+* [DST contents](/docs/27004)
+
+## Source code
+The sources are available on https://gitlab.cern.ch/delphi. Some modules still requires CERN authentication. The plan is to release the software in the near future.
+
+## Compilers:
+The DELPHI stack is mostly written in Fortran, with some bits written in C. Only gfortran and GNU gcc are supported. We use the gfortran version which comes with the supported operating system.
+
+## Getting help
+
+The collaboration main contact for data preservation is the mailing list DELPHI-data-preservation-board@cern.ch. Support can only be given on a best effort basis. Suggestions and feedback is of course welcome!
+
+# Examples
```

Comment:
BTW I have a feeling that the page is getting long. It is OK to have one getting started page for everything, but then perhaps we should put a table on contents at the top to simplify user navigation? See for example https://opendata.cern.ch/docs/cms-guide-for-condition-database

(The idea is to guide readers more easily between various sections if the page seems too long and they may "loose" the drive.)

An alternative would be to create more help pages, a short Getting Started one, then some more detailed pages about dedicated topics such as raw data access, running MC simulations, _etc_ so that one page would have strictly one topic.

Reply:
Giving it a shot with TOC